SECURITY.md

```diff
@@ -28,10 +28,11 @@ release, and disclosure processes, as well as general security posture.
 The following versions are currently supported and receive security updates.
 Release candidates will not receive security updates.
 
-| Version  | Supported          |
-| -------- | ------------------ |
-| >= 1.4.x | :white_check_mark: |
-| <=1.3.x  | :x:                |
+| Version   | Supported          |
+| --------- | ------------------ |
+| >= 2.0.x  | :white_check_mark: |
+| >= 1.10.x | :white_check_mark: |
+| <=1.3.x   | :x:                |
```

Comment:
The current edit seems to miss the releases between 1.3.x and 1.10.x ?
```suggestion
| <=1.9.x   | :x:                |
```

Reply:
Yes. Done.